bin/_mocha

```diff
@@ -323,9 +323,25 @@ var args = program.args;
 if (!args.length) args.push('test');
 
 args.forEach(function(arg){
-  files = files.concat(utils.lookupFiles(arg, extensions, program.recursive));
+  var newFiles;
+  try {
+    newFiles = utils.lookupFiles(arg, extensions, program.recursive);
+  } catch(err) {
+    if (err.message.indexOf('cannot resolve path') === 0) {
+      console.error('Warning: Could not find any test files matching pattern: ' + arg);
+      return;
+    } else {
+      throw err;
+    }
+  }
+
+  files = files.concat(newFiles);
 });
 
+if (files.length === 0) {
+  process.exit(1);
```

Comment:
Should probably add an error message here about no files being found, given you can simply execute `mocha` in a directory with empty subdir `test`, and this is the current output:

```shell
$ mocha


  0 passing (3ms)

```